drivers/vdpa/virtio/virtio_vdpa.h

```diff
@@ -0,0 +1,56 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2022 NVIDIA Corporation & Affiliates
+ */
+
+#ifndef _VIRTIO_VDPA_H_
+#define _VIRTIO_VDPA_H_
+
+enum {
+	VIRTIO_VDPA_NOTIFIER_STATE_DISABLED,
+	VIRTIO_VDPA_NOTIFIER_STATE_ENABLED,
+	VIRTIO_VDPA_NOTIFIER_STATE_ERR
+};
+
+struct virtio_vdpa_vring_info {
+	uint64_t desc;
+	uint64_t avail;
+	uint64_t used;
+	uint16_t size;
+	uint16_t index;
+	uint8_t notifier_state;
+	bool enable;
+	struct rte_intr_handle *intr_handle;
+	struct virtio_vdpa_priv *priv;
+};
+
+#define VIRTIO_VDPA_DRIVER_NAME vdpa_virtio
```

Comment:
string should be in "vdpa_virtio".

Reply:
current in DPDK, all driver don't use "" in RTE_PMD_REGISTER_PCI